custom/enikshay/tests/test_adherence_updates.py

```diff
@@ -671,3 +671,53 @@ def test_update_by_person(self):
             {key: episode.dynamic_case_properties()[key] for key in expected_update},
             {key: str(val) for key, val in expected_update.iteritems()}  # convert values to strings
         )
+
+    def test_adherence_score_start_date_month(self):
+        # If the start date is more than a month ago, calculate the last month's scores
+        self.case_updater.date_today_in_india = datetime.date(2016, 1, 31)
+        self.assert_update(
```

Comment:
I totally agree, but didn't have time to update all the other test cases. It was very confusing to write them.